src/iframe-helper.js

```diff
@@ -158,6 +158,15 @@ function isDescendantWindow(ancestor, descendant) {
   return false;
 }
 
+export function isInIframe(element) {
+  dev().assert(element);
+  const win = element.ownerDocument.defaultView;
+  if (win && win != win.parent) {
```

Comment:
Updated. updated the code. Kept the condition separate as it is easier to read. lmk if you would still like this to change.